sites/cheerpx/src/content/docs/12-reference/13-CheerpX-Linux-setKmsCanvas.md

```diff
@@ -0,0 +1,61 @@
+---
+title: CheerpX.Linux#setKmsCanvas
+Description: Configure the KMS canvas for rendering graphical output in a CheerpX Linux instance.
+---
+
+```ts
+namespace CheerpX {
+	class Linux {
+		setKmsCanvas(
+			canvas: HTMLCanvasElement,
+			width: number,
+			height: number
+		): void;
+	}
+}
+```
+
+## Parameters
+
+- **canvas: HTMLCanvasElement** - The `canvas` element in your HTML document that serves as the rendering surface. This is where graphical output is displayed.
+
+- **width: number** - The width of the canvas in pixels.
```

Comment:
```suggestion
- **width: number** - The desired width of the canvas in pixels.
```

The idea is that CheerpX will internally resize the pixel when Xorg takes control of the display